packages/@aws-cdk/cdk/README.md

```diff
@@ -3,3 +3,149 @@
+|MarketingSystem/MarketingVpc/PublicSubnet1| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PublicSubnet1/RouteTable| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PublicSubnet1/NATGateway| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PublicSubnet2| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PublicSubnet2/RouteTable| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PublicSubnet2/NATGateway| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PublicSubnet3| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PublicSubnet3/RouteTable| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PublicSubnet3/NATGateway| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PrivateSubnet1| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PrivateSubnet1/RouteTable| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PrivateSubnet2| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PrivateSubnet2/RouteTable| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PrivateSubnet3| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/PrivateSubnet3/RouteTable| CostCenter | Platform|
+|MarketingSystem/MarketingVpc/IGW|CostCenter|Platform|
+|MarketingSystem/B2BService/Service/SecurityGroup/Resource|CostCenter|Marketing|
+|MarketingSystem/B2BService/LB/Resource|CostCenter|Marketing|
+|MarketingSystem/B2BService/LB/SecurityGroup/Resource|CostCenter|Marketing|
+|MarketingSystem/B2BService/LB/PublicListener/ECSGroup/Resource|CostCenter|Marketing|
+|MarketingSystem/B2CService/Service/SecurityGroup/Resource|CostCenter|Marketing|
+|MarketingSystem/B2CService/LB/Resource|CostCenter|Marketing|
+|MarketingSystem/B2CService/LB/SecurityGroup/Resource|CostCenter|Marketing|
+|MarketingSystem/B2CService/LB/PublicListener/ECSGroup/Resource|CostCenter|Marketing|
+
+As you can see many tags are generated with only a few intent based directives. The CDK does default some additional tags for suggested `Name` keys. If you want to remove those tags you can do so by using the `RemoveTag` aspect, see below:
+
+```ts
+// snip //
+const vpc = new ec2.VpcNetwork(marketingStack, 'MarketingVpc', {
+  maxAZs: 3 // Default is all AZs in region
+  });
+// override the VPC tags with Platform
+// this will tag the VPC, Subnets, Route Tables, IGW, and NatGWs
+vpc.apply(new cdk.Tag(COST_CENTER_KEY, 'Platform'));
+vpc.apply(new cdk.RemoveTag('Name'));
+// snip //
+```
+
+This will remove the name tags from the VPC, subnets, route tables and NAT
+gateways. If you've been following closely, this may lead you to ask how does
+remove work when the tag is actually applied closer to the resource? The Tag API
+has a few features that are covered later to explain how this works.
+
+### API
+
+In order to enable additional controls a Tags can specifically include or
+exclude a CloudFormation Resource Type, propagate tags for an autoscaling group,
+and use priority to override the default precedence. See the `TagAspectProps` 
```

Comment:
This should be called `TagProps`, no?

Reply:
Also show a quick snippet on how to use these options

Reply:
What do you think if I just move the `includeResourceType` and exclude to the `TagProps` interface. Then have `TagManager` own all the logic for applying the tag to the resource. Right now it's split because `priority` had to managed at the `TagManager` and include/exclude was done in the `visit` logic. I think eliminating `TagAspectProps` moving all this to `TagProps` is the right choice.

Reply:
Yes, makes sense